sql/catalyst/src/main/scala/org/apache/spark/sql/Encoders.scala

```diff
@@ -27,6 +27,7 @@ import org.apache.spark.sql.catalyst.encoders.{encoderFor, ExpressionEncoder}
 import org.apache.spark.sql.catalyst.expressions.{BoundReference, Cast}
 import org.apache.spark.sql.catalyst.expressions.objects.{DecodeUsingSerializer, EncodeUsingSerializer}
 import org.apache.spark.sql.types._
+import org.apache.spark.unsafe.types.CalendarInterval
```

Comment:
Why don't we move `CalendarInterval` to `org.apache.spark.types` before we more publicly expose this?

Reply:
If we're not sure whether to make this as a proper public type or not, let's don't add it for now.